line-items.go

```diff
@@ -48,7 +50,7 @@ type Property struct {
 // GetPropertyByName gets a line item property by its name
 func (lineItem LineItem) GetPropertyByName(name string) Property {
 	for _, property := range lineItem.Properties {
-		if property.Name == name {
+		if property.Name == name || property.Name == fmt.Sprintf("_%s", name) {
```

Comment:
I think its better to be specific on what property name you are getting rather than manipulate what the user is asking for i.e. call it twice.If there is a concept as hidden properties then it should be a different method.

Reply:
They're only really hidden from the customer on the checkout.
Also this really just a small patch. Does this `getPropertyByName` get used anywhere else? Maybe the Patient API?

Reply:
Anybody could use this package though. We are in effect adding a fix in here for us so it works for V1 and V2 however we could have hundreds of services using this Shopify package that's not even related to this problem.

Reply:
What if someone wanted to get a property name but disregard underscores?

Reply:
This could be my opinion, but having an `_` in a line item in Shopify means that the merchant would like to hide it. 

I can't see any reason why a line item would have both `_customProperty` and `customProperty` under the same line item at the same time with different values. 

An alternative solution would be to pass a parameter in this function as follows:
```
func (lineItem LineItem) GetPropertyByName(name string, includeHidden) Property {
	for _, property := range lineItem.Properties {
		if property.Name == name || (includeHidden && property.Name == fmt.Sprintf("_%s", name)) {
			return property
	    	}
	}
	return Property{}
}

Reply:
Seems ok considering it's probably not going to have both. A link to some docs to explain the hidden property would be nice as well.

Reply:
Only thing also is that now you need to specify that param for every call because golang does not allow. Thats why it would be nicer for it to have a different function.

Reply:
If anyone upgrade the version, then it would need to add that parameter 🙂 

Having a different function would mean that if someone would like to find a property, it'll need to call two functions (and loop through the properties twice)

Reply:
Yeh but they would know whether it was hidden or not.

Reply:
Not always - in our case we didn't know and it caused issues 🙂